sefaria/settings.py

```diff
@@ -330,7 +330,9 @@
 
 
 if SENTRY_DSN:
-    init_sentry(SENTRY_DSN)
+    SENTRY_CODE_VERSION = os.getenv("SENTRY_CODE_VERSION", "unknown")
+    SENTRY_ENVIRONMENT = os.getenv("SENTRY_ENVIRONMENT", "unknown")
```

Comment:
We still are left with a problem here where this is still creating a new pattern where local developers are required to define environment variables, something not required before. 

Perhaps this entire function call should be in local_settings rather than settings (since it is something that changes in between environments)